inc/tagitem.class.php

```diff
@@ -419,7 +419,8 @@ public static function updateItem(CommonDBTM $item, bool $delete_existing_tags =
         $tag      = new PluginTagTag();
         $tag_item = new self();
 
-        if (!$tag::canUpdate()) {
+        // Be careful to not check right if the change is coming from the cron
+        if (!isCommandLine() && !$tag::canUpdate()) {
```

Comment:
Con may be executed in web mode.
```suggestion
        if (!Session::isCron() && !$tag::canUpdate()) {
```